audit/projects/kubernetes-public/buckets/k8s-triage/iam.json

```diff
@@ -22,16 +22,14 @@
     },
     {
       "members": [
-        "serviceAccount:k8s-triage@k8s-infra-prow-build-trusted.iam.gserviceaccount.com",
-        "serviceAccount:triage@k8s-gubernator.iam.gserviceaccount.com"
```

Comment:
nit: I'd revert this file's changes, I don't think the "audit/" files should be manually updated.